spanner/api/Spanner.Samples.Tests/ListBackupOperationsTest.cs

```diff
@@ -25,11 +24,12 @@ public ListBackupOperationsTest(SpannerFixture spannerFixture)
         _spannerFixture = spannerFixture;
     }
 
-    [Fact(Skip = "https://github.com/GoogleCloudPlatform/dotnet-docs-samples/issues/1583")]
+    [Fact]
     public void TestListBackupOperations()
     {
         ListBackupOperationsSample getBackupOperationsSample = new ListBackupOperationsSample();
-        var operations = getBackupOperationsSample.ListBackupOperations(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.BackupDatabaseId).ToList();
-        Assert.True(operations.Count >= 0);
+        var operations = getBackupOperationsSample.ListBackupOperations(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.BackupDatabaseId);
+        // We iterated over operation on the sample itself, let's not do that again.
```

Comment:
Is this actually testing anything useful then?

Reply:
(Same question for tests below.)

Reply:
Just that the method is returning. I had it at Assert.NotEmpty but it still reached quota.

I could just call the method and have the test green if the method does not fail, but I'm hoping we can do better in the future so I left everything ready.